rfc/tableAvailabilityMarkdown.md

```diff
@@ -0,0 +1,108 @@
+# Table resource supporting filtered availability
+
+This RFC implements [Github Issue 306](https://github.com/yahoo/fili/issues/306)
+
+## Goals
+
+1. Show the queryable range of time as part of the Logical Table endpoint view.
+2. Allow the queryable range to reflect constraints as would be found in a data request.
+  * Required metrics and dimensions
+  * Applied filters
```

Comment:
The filters can impact availability if dimension /values/ impact the query planning.  (right now that's only a concern on physical tables with dimension value partitions)